### PR TITLE
Update node spec

### PR DESCRIPTION
**Pull request by dalvizu**
**Description**:
Updated node spec, reflecting hardware refresh in 2023, as approved by techcom. Spec is ~5 years old at this point.

Note: This is a draft pending final performance testing results

### CODE REVIEW

**Review by theekrystallee — APPROVED** [review]
Didn't mean to send this here as approved

**Comment by SimiHunjan** [comment]
@dalvizu was this PR not meant to be in "Draft" state?

**Comment by SimiHunjan** [comment]
NVM saw your comment in the description regarding intent for draft

File: networks/mainnet/mainnet-nodes/node-requirements/README.md [hunk 1]
```diff
@@ -8,44 +8,49 @@ description: >-
...
+      - Recommended: 2800 or greater
+* Network Connectivity: Sustained 1Gb/s internet bandwidth via a single 1-Gigabit / 10-Gigabit Ethernet interface
+* Memory: 256 GB PC4-21300 2666MHz DDR4 ECC Registered DIMM or faster (minimum), 320GB or higher PC4-25600 3200MHz (recommended)
+* Storage: It is recommended to mount 240 GB SSD with Raid 1 as a root volume `/` and then provide usable storage via different
+  devices later mounted during installation. This may not be possible on your hardware, so alternatively all required 
+  storage may be allocated to the root volume.
+  - Minimum: 5TB of SSD NVMe usable storage
+  - Recommended:
+    - 2 x 240GB SSD with RAID 1 for OS Storage
+    - 2 x NVMe devices as a 7.5TB RAID 0 (or 4x as RAID 10 array)
```
Comment (ElijahLynn) [inline networks/mainnet/mainnet-nodes/node-requirements/README.md:34]: TODO: Add info about filesystem, e.g. ext4 I think.

Comment (dalvizu) [reply networks/mainnet/mainnet-nodes/node-requirements/README.md:34]: That is out of scope: ext4 is covered in the installation process. This guide is intended to guide hardware purchase / provisioning requirements.

**Review by theekrystallee — APPROVED** [review]
LGTM. Thanks for the PR!

**Pull request merged**